augur/filter/__init__.py

```diff
@@ -3,11 +3,9 @@
 """
 from augur.dates import numeric_date_type, SUPPORTED_DATE_HELP_TEXT
 from augur.types import EmptyOutputReportingMethod
+from . import config
```

Comment:
I would call this `constants` not `config`. Config implies it's changeable, particularly by the user or system. The values going in here are constants.

Reply:
Updated in [force-push](https://github.com/nextstrain/augur/compare/6a8630541862a672e37c6e440c010a6dcbc3b607..6050e8d9571fe0574f331466a0dae26cf0890816).